bundles/org.openhab.binding.miio/src/main/resources/OH-INF/i18n/miio.properties

```diff
@@ -155,7 +181,14 @@ channel-type.miio.water_box_carriage_status.label = Water Box Carriage State
 channel-type.miio.water_box_mode.label = Water Box Mode
 channel-type.miio.water_box_status.label = Water Box State
 
-# Thing status descriptions
+# channel types
+
+channel-type.miio.cleanmopstart.label = Start mop wash
+channel-type.miio.cleanmopstop.label = Stop mop wash
+channel-type.miio.collectdust.label = Start collecting dust at the station
+
```

Comment:
These needs to be manually removed since they don't exist anymore. The tool will only move them, but not delete them, since it can't know for sure if they are referenced dynamically.

Reply:
@jlaur  I removed them